my-sms/db/schema.rb

```diff
@@ -23,6 +22,13 @@
     t.string   "gender"
     t.datetime "created_at",  :null => false
     t.datetime "updated_at",  :null => false
+    t.integer  "title_id"
+  end
+
+  create_table "titles", :force => true do |t|
+    t.string   "name"
```

Comment:
perhaps non null too?

Reply:
Fixed in 14c14305da823123d7a32351cae62a6dcab11775